src/Algebra/Apartness/Bundles.agda

```diff
@@ -36,8 +36,8 @@ record HeytingCommutativeRing c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ
   commutativeRing : CommutativeRing c ℓ₁
   commutativeRing = record { isCommutativeRing = isCommutativeRing }
 
-  apartnessRelation : ApartnessRelation c ℓ₁ ℓ₂
-  apartnessRelation = record { isApartnessRelation = isApartnessRelation }
```

Comment:
Is `apartnessRelation` still exported from somewhere? I can't spot it. If not, we should reopen `tightApartnessRelation`?

Reply:
Ah... I slightly screwed up the latest round of commits wrt the merge conflict arising from #2576 so will do another round of checking...